node/src/types/block.rs

```diff
@@ -1233,6 +1220,17 @@ pub(crate) mod json_compatibility {
     }
 
     impl JsonBlock {
+        /// Create a new JSON Block with a Linear chain block and its associated signatures.
+        pub fn new(block: Block, signatures: BlockSignatures) -> Self {
+            let mut proto_block = JsonBlock::from(block);
+            proto_block.proofs = signatures
+                .proofs
+                .iter()
```

Comment:
If we use `into_iter` there would be no need to copy the values in `JsonProof::new()`. 

In fact we could simplify this to just `signatures.proofs.into_iter().map(JsonProof::from).collect();`.

Reply:
Used suggestiong